lib/sinon/default-behaviors.js

```diff
@@ -181,6 +183,12 @@ module.exports = {
         return fake;
     },
 
+    resolvesThis: function resolvesThis(fake) {
+        fake.resolveThis = true;
+        fake.resolve = false;
+        fake.reject = false;
```

Comment:
@fatso83 I think that link only works inside the files tab. When I click it inside of it the link works.

Basically I think the following flags should be set:

* `fake.returnValue`
* `fake.returnValueDefined`
* `fake.exception`